### ISSUE TITLE
MFSearchBuilder.FindCount extension method

### ISSUE THREAD

**Issue by CraigHawker** [M-Files/VAF.Extensions.Community#27]
Would be a shorthand for https://www.m-files.com/api/documentation/#MFilesAPI~VaultObjectSearchOperations~GetObjectCountInSearch.html.

**Comment by aaron-hardin** [comment]
We do already have this: https://github.com/M-Files/VAF.Extensions.Community/blob/994fa9e0f9d00f8b85ddedfaeada0557195b52dd/MFiles.VAF.Extensions/ExtensionMethods/MFSearchBuilderExtensionMethods/SegmentedSearch.cs#L133

I guess in a lot of cases the segmented part of it could slow down the search though.

**Comment by CraigHawker** [comment]
This is true, but the segmented search may indeed be quite an overhead if the matching objects have very diverse IDs.
